mono/utils/mono-state.c

```diff
@@ -148,6 +148,15 @@ file_for_summary_stage (const char *directory, MonoSummaryStage stage, gchar *bu
 	g_snprintf (buff, sizeof_buff, "%s%scrash_stage_%d", directory, G_DIR_SEPARATOR_S, stage);
 }
 
+static void
+create_stage_mark_file (void)
+{
+	char out_file [200];
+	file_for_summary_stage (log.directory, log.level, out_file, sizeof(out_file));
+	int handle = g_open (out_file, O_WRONLY | O_CREAT, S_IWUSR | S_IRUSR | S_IRGRP | S_IROTH);
+	close(handle);
```

Comment:
Coverity didn't like that we call `close (handle)` without checking that `handle > 0`.

```
>>>     CID 1454754:  Error handling issues  (NEGATIVE_RETURNS)
>>>     "handle" is passed to a parameter that cannot be negative.
```

Reply:
I'll push a fix. That pattern was already there though, is Coverity only looking at diffs?

Reply:
The website has a report for the whole codebase,  but it only sends me emails about changed code.